index.html

```diff
@@ -424,6 +444,84 @@ <h2 class="experience-text">Skills</h2>
           </li>
         </ul>
       </section>
+
+      <!-- Contact 4/4 - Layout -->
+
+      <section class="l-contact">
+        <!-- Background Images -->
+
+        <div class="e-12"></div>
+        <div class="e-3"></div>
+        <div class="e-5"></div>
+        <div class="r-7"></div>
+        <div class="r-7 r-7-b"></div>
+
+        <!-- Block 1/3 - Text -->
+
+        <div class="l-contact__text">
+          <h2 class="text-title">Contact me</h2>
+          <p class="text-parraf">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <!-- Block 2/3 - Form -->
+
+        <form action="https://formspree.io/f/mwkaodba" method="post">
+          <ul>
+            <li>
```

Comment:
- [ ] **OPTIONAL** In the `<form>` tag, the browser knows that the items within this tag belong to it and they're listed already. So, it doesn't make sense to use `<ul>` and `<li>` tags to sort them as a list of items. It'd be nicer if you managed that!
_Consider putting each input item inside a `<div>` tag._

Reply:
Hi Mahmoud! Thanks for your review.  I will do the changes you suggest.
However, I have a doubt.
Could you please refer me to a document where I can see the criteria you all, the code reviewers, follow to evaluate the code? I am getting confused with the contradictory suggestions from different code reviewers, and I would like to know if there is a single source of truth to follow and help me to write better code.
 Last week a code reviewer told me about using ul tag elements always that I need to display a list of elements, and yesterday I have approved a similar exercise ( the portfolio of my coding partner in the pair programming exercise ), where we used ul and li tags. The code got approved without any comment from the code reviewer and now I'm really confused.
Please, don't take this comment as a critique, my intention is just to improve my code.
Have a nice day and thanks for your time.
Abraham

Reply:
Hi @Metsanpeitto
Thank you for reaching out, and don't worry about anything. In fact, I'm glad you are communicating about your doubts because that's the main point of the review. 
Regarding your concerns, this is an optional request (from my perspective) as a software engineer and a code reviewer. Although, me and other code reviewers must only ask students for what they already have experienced (which is the project requirements), and amy other requests may come from personal experience or general knowledge (as something common between most of the developers).
Regarding the specific point, and as I explained in the comment, `ul` stands for `unordered list` and `li` stands for `list item`. Following the Semantic HTML, there should not be one tag without the other, and they exist to help making lists. So, think about it! Do you think that the form's inputs can be considered list items inside a list? Can we actually call a form, a list? They're two different elements (tags) right?
The point is, Semantic meanings of HTML have standard concepts that we're all agreeing to, but as long as there's something called development, there'll always be variations, and here we think to come up with the best solution for our situation.
And for your request, you can go to the [`form`](https://www.w3schools.com/tags/tag_form.asp) tag and read a little about it. Also, you can hit any framework and see how they combine the form's elements together.
[Bootstrap](https://getbootstrap.com/docs/4.0/components/forms/) and [Tailwind](https://v1.tailwindcss.com/components/forms )
And of course you can look at the lists as well.
I hope I answered your questions and doubts. 
Please reply here if you still have any! 😊

Reply:
Thanks, this was really helpful and will help me a lot in the future.